.github/workflows/kind_multicluster_e2e_tests.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         e2e_test:
           - CreateMultiDatacenterCluster
           - CreateMixedMultiDataCenterCluster
-          - AddDcToCluster
+          - AddDcToClusterDiffDataplane
```

Comment:
This test was renamed because we have another test with the `AddDcToCluster` prefix (`AddDcToClusterSameDataplane`), which was executed each time `AddDcToCluster` was invoked.
This is because subtests in Go seem to execute every test that starts with the configured value.